.buildkite/premerge.steps.yaml

```diff
@@ -17,6 +17,23 @@ script_runner: &script_runner
   - "scaler_version=2"
   - "working_hours_time_zone=london"
 
+common: &common
+  agents:
+    - "agent_count=1"
+    - "capable_of_building=gdk-for-unreal"
+    - "environment=production"
+    - "machine_type=quad-high-cpu"  # this name matches to SpatialOS node-size names
+    - "platform=windows"
+    - "permission_set=builder"
+    - "scaler_version=2"
+    - "queue=${CI_WINDOWS_BUILDER_QUEUE:-v4-2019-11-11-bk3891-f17bd2c787d2fe1c}"
```

Comment:
Where does this constant come from? Does it change frequently?

Reply:
It is the queue corresponding to a particular built agent image. It's supplied in Buildkite when that agent image is built (e.g. https://buildkite.com/improbable/platform-bakery/builds/4343). Technically it shouldn't update often, but due to recent buildkite changes a couple of image changes have been required to make scheduled (nightly) builds work.